keps/sig-storage/2924-csi-migration-cephfs/kep.yaml

```diff
@@ -12,9 +12,9 @@ approvers:
   - "@msau42"
 editor: "@Jiawei0227"
 creation-date: 2022-01-27
-last-updated: 2022-01-27
+last-updated: 2023-05-28
 disable-supported: true
-status: implementable
+status: withdrawn
```

Comment:
I'm not sure if "withdrawn" is a valid KEP status.  @msau42 ?

Reply:
FYI:

NNNN-kep-template

https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L9

```
status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
```